HW06.Rmd

```diff
@@ -119,19 +125,20 @@ H3 <- GBM.genes.of.interest %>%
   gather(key = "file", value = "value", 2:24) %>%
   spread(key = gene_symbol, value = value)
 
-a <- colnames(WT)
+gene_list <- colnames(WT) #We need a vector with all the gene names. Careful, this will include the "file" column, as well. 
 
-tests <- list()
-tests_table <- matrix(nrow = 27, ncol = 2, data = NA)
+tests <- list() #Empty list to work with
+tests_table <- matrix(nrow = 27, ncol = 3, data = NA) #Empty data frame to work with
 
-for(i in 2:ncol(H3)) {
-  tests[[a[i]]] <- t.test(H3[,i],WT[,i])
-  gene <- a[i]
-  p.value <- round(tests[[i-1]][[3]], digits = 6)
-  tests_table[i-1,] <- c(gene, p.value)
+for(i in 2:ncol(H3)) { 

```

Comment:
I found the logic of this sort of difficult to follow, but it works well! It's always neat to see how problems can be solved in different ways